docker/docker-compose.yml

```diff
@@ -33,10 +33,13 @@ services:
 
   nodejs:
     image: fidals/se-nodejs:dev
-    volumes_from:
-      - app
+    # For refarm-site's front development
+    # volumes:
+    #  - $REFARM_DIR:/usr/app/refarm-site
     env_file:
       - env_files/paths
+    environment:
+      - DEPS_DIR=/usr/app/refarm-site
```

Comment:
We are storing a `DEPS_DIR` in `env_files/paths`, so this seems redundant

Reply:
Done.